scalameta/trees/src/main/scala/scala/meta/internal/prettyprinters/TreeStructure.scala

```diff
@@ -56,7 +56,13 @@ object TreeStructure {
             case Ident("-") => true
             case _ => false
           }
-          s(x.tokens.filter(isRelevantToken _).map(_.show[Syntax]).mkString)
+          def showToken(tok: Token) = tok match {
+            case Constant.Long(v) => Show.Long(v.longValue)
+            case Constant.Float(v) => Show.Float(v.floatValue)
+            case Constant.Double(v) => Show.Double(v.doubleValue)
```

Comment:
I suggest we convert these constants to `Show.Str` in order to keep the `Show` framework as simple as possible.